pkg/netpol/connlist/exposure_analysis.go

```diff
@@ -4,27 +4,17 @@ import (
 	"github.com/np-guard/netpol-analyzer/pkg/netpol/internal/common"
 )
 
-// exposureMap from peer to its exposure data; map that stores refined exposure-analysis allowed connections
-// which are computed by the connlist analyzer
-type exposureMap map[Peer]*peerExposureData
-
-// following const implements an enum type for the initial value of a peer's
-// ingressProtected or egressProtected when adding a new entry in the map
-// a peer is protected on a direction (ingress/egress) if there is at least one network-policy
-// (that affects the direction) selecting it;
-// the final value on the map should be protected or notProtected only
-const (
-	unknown = iota
-	protected
-	notProtected
-)
+// exposureMaps is a struct containing two maps from peer to its exposure data, one for each direction ingress/egress;
+// maps that store refined exposure-analysis allowed connections which are computed by the connlist analyzer
+type exposureMaps struct {
+	ingressExposureMap map[Peer]*peerXgressExposureData
+	egressExposureMap  map[Peer]*peerXgressExposureData
+}
 
-// peerExposureData stores exposure data for a peer
-type peerExposureData struct {
-	isIngressProtected int
-	isEgressProtected  int
-	ingressExposure    []*xgressExposure
-	egressExposure     []*xgressExposure
+// peerXgressExposureData store exposure data of a peer on one direction ingress/egress
+type peerXgressExposureData struct {
+	isProtected  bool
+	exposureInfo []*xgressExposure
```

Comment:
can you add documentation on the fields here and of type `xgressExposure` ?
what each field means, and possible relations between fields values. 
for example, if `isProtected` is false, is `exposureInfo` expected to be empty or not .. etc

Reply:
done